geotrek/maintenance/filters.py

```diff
@@ -23,18 +24,17 @@ def get_geom(self, value):
     def filter(self, qs, values):
         if not values:
             return qs
-        if not isinstance(values, list):
-            values = [values]
+        if isinstance(values, list):
+            bbox = GeometryCollection([self.get_geom(value) for value in values])
```

Comment:
```suggestion
            bbox = GeometryCollection([self.get_geom(value) for value in values])
```
I think it's not a bbox in this case